ios/Wrappers/RNWrappers.m

```diff
@@ -412,8 +412,12 @@ + (JPRecommendationConfiguration *)recommendationConfigurationFromConfiguration:
     NSURL *URL = [NSURL URLWithString:URLString];
     NSString *RSAPublicKey = [dictionary stringForKey:@"rsaPublicKey"];
     NSNumber *timeout = [dictionary optionalNumberForKey:@"timeout"];
+    NSNumber *haltTransactionInCaseOfAnyError = [dictionary optionalBoolForKey:@"haltTransactionInCaseOfAnyError"];
 
```

Comment:
Is it necessary that we receive haltTransactionInCaseOfAnyError as NSNumber, instead of bool straight away?

Reply:
That's an `NSDictionary`; you can only have Objects in dictionaries. Hence, that's why we have an `NSNumber` instance there.

Reply:
oh right, makes sense, good to know